site/content/en/docs/user/kwokctl-snapshot/etcd-snapshot.demo

```diff
@@ -0,0 +1,29 @@
+# Let's getting started with kwokctl!
+kwokctl create cluster
+
+# Create an nodes.
+kwokctl scale node --replicas 2
+
+# Apply a deployment.
```

Comment:
The demo looks great. However, the user needs to switch contexts before applying a deployment. For beginners new to Kubernetes, this can be confusing, especially if they have been working with a cluster that has a different name beforehand.

**Suggestion**
```
# Use context
kubectl cluster-info --context kwok-kwok
```

Reply:
But that doesn't switch context, it needs uses:

``` bash
kubectl config use-context kwok-kwok
```

and I feel it is enough to add to the demo of the home page 
https://kwok.sigs.k8s.io/#getting-started

WDYT?

Reply:
Oh, it's true. I just realized that it only shows details about the DNS name the control plane is running at.
For example:
```
kubectl cluster-info --context kwok-kwok
Kubernetes control plane is running at https://127.0.0.1:32766

To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
```

Reply:
I also noticed that after creating a cluster using kwokctl, it automatically switches the context.
For example:
```
$ kwokctl create cluster
Cluster is creating                                                                                                       cluster=kwok
Cluster is created                                                                                           elapsed=0.9s cluster=kwok
Cluster is starting                                                                                                       cluster=kwok
Cluster is started                                                                                           elapsed=1.9s cluster=kwok
You can now use your cluster with:

        kubectl cluster-info --context kwok-kwok

Thanks for using kwok!

$ kubectl config current-context
kwok-kwok
```

Reply:
It means there is no point in adding this suggestion:

```
# Use context 
kubectl cluster-info --context kwok-kwok
```

Reply:
> My sincere apologies. After executing kwokctl create cluster, this statement You can now use your cluster with: confused me.

It's modeled on kind. 
https://github.com/kubernetes-sigs/kind/blob/bbcc02f84c8efd68d20c3a6585c2f282d7c1b8c9/pkg/cluster/internal/create/create.go#L187-L197

Reply:
True, I surprisingly saw it yesterday while creating a cluster using Kind.
Also, if no cluster name is specified when creating a Kind cluster, it switches to a default context after creating it. It's nice that the same applies to kwok too.